### ISSUE TITLE
5.x on JRuby: Latest Changes throws TypeError

### ISSUE THREAD

**Issue by bartkamphorst** [gollum/gollum#1431]
Trying to view the `latest_changes` pages throws the following error on JRuby: 
```
2019-10-02 09:32:21 - TypeError - no implicit conversion of Symbol into Integer:
	org/jruby/RubyArray.java:1458:in `[]'
	[...]/gollum-lib_rjgit_adapter/lib/rjgit_adapter/git_layer_rjgit.rb:139:in `stats'
	[...]/gollum/lib/gollum/views/latest_changes.rb:25:in `block in versions'
	org/jruby/RubyArray.java:2486:in `map'
	[...]/gollum/lib/gollum/views/latest_changes.rb:15:in `versions'
	org/jruby/RubyMethod.java:111:in `call'
```

**Comment by bartkamphorst** [comment]
I'm seeing the same error in one of gollum-lib's tests:
```
Error: test_get_existing_page(Page): TypeError: no implicit conversion of Symbol into Integer
org/jruby/RubyArray.java:1458:in `[]'
[...]/gollum-lib_rjgit_adapter/lib/rjgit_adapter/git_layer_rjgit.rb:139:in `stats'
[...]/gollum-lib/test/test_page.rb:27:in `block in test_get_existing_page'

**Comment by dometto** [comment]
Could it be that you were accidentally still using an older version of the adapter (perhaps due to local edits to your `Gemfile`)? I can't reproduce this in neither gollum nor gollum-lib. The travis tests also seem to be passing.

**Comment by bartkamphorst** [comment]
Fixed by updating to latest RJGit version (5.5.0.0) with bundle update.
